config/locales/es.yml

```diff
@@ -2560,6 +2560,20 @@ es:
         edit:
           helper_description: Continúe viendo para qué más podemos ayudarlo a calificar.
           title: Parece que no eres elegible para este crédito.
+      nj_medical_expenses:
+        edit:
+          help_text: If you do not want to claim this deduction, click “Continue”.
```

Comment:
[SAND] Is it possible to stick all this in Google Translate? I've been doing that for all my placeholder Spanish text. Does it matter, since we're replacing it in the future anyway?

Reply:
Per our slack conversation, leaving this for now. Added this to the translation ticket in the backlog